openassessment/xblock/openassessmentblock.py

```diff
@@ -1154,25 +1155,40 @@ def publish_event(self, data, suffix=''):  # pylint: disable=unused-argument
         self.runtime.publish(self, event_name, data)
         return {'success': True}
 
-    def get_username(self, anonymous_user_id):
+    def get_real_user(self, anonymous_user_id):
```

Comment:
nit: why is there "real" in get_real_user ? Why not call it get_user ?

Reply:
"real" as opposed to "anonymous", plus it passes through to `self.xmodule_runtime.get_real_user`, so it keeps the naming consistent.